platform/ios/MGLMapView.mm

```diff
@@ -142,6 +142,7 @@ - (instancetype)initWithCoder:(NSCoder *)decoder
     if (self && [self commonInit])
     {
         self.styleURL = nil;
+        self.accessToken = [MGLAccountManager accessToken];
```

Comment:
Note that this calls through to `-setAccessToken:`, which calls through to `Map::setAccessToken()` unconditionally. I think it should be fine, though.